lib/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsSqsConfiguration.kt

```diff
@@ -9,9 +9,9 @@ import org.springframework.context.annotation.DependsOn
 import org.springframework.jms.annotation.EnableJms
 
 @Configuration
-@EnableConfigurationProperties(HmppsQueueProperties::class)
+@EnableConfigurationProperties(HmppsSqsProperties::class)
 @EnableJms
-class HmppsQueueConfiguration {
+class HmppsSqsConfiguration {
```

Comment:
Renamed class to match library name.  However all components that trade in HmppsQueue instances are still prefixed HmppsQueue.